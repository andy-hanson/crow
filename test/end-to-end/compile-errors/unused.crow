import
	# unused fun
	crow/io/duration: sec
	# unused type
	crow/io/time-format: time-locale
	# unused type alias
	system/unistd: ssize_t
	# unused spec
	crow/col/collection: begin-end
	# unused module
	crow/io/process

main void()
	()

foo[t] void(p nat, _ string)
	_ act void(nat) = x => ()
	_ act void(nat) = _ => ()
	y = 2
	()

-bar[t] void() t compare
	()

-r record mut
	r0 -mut nat

-u union
	u0 nat

-a alias
	nat

-e enum
	e0

-f flags
	f0

-s spec
	foo void()
