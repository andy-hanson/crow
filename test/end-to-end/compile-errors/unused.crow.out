exit code: 0
stderr:
test/end-to-end/compile-errors/unused.crow 26:8-26:9 Field 'r0' mutability would already be private by default.
